test/e2e/network_segmentation.go

```diff
@@ -1057,20 +1057,21 @@ spec:
 				}
 				Expect(found).To(BeTrue(), "NetworkAllocationSucceeded condition should be True when cluster has <= 2 nodes")
 			} else {
-				By("when cluster has > 2 nodes, error is expected")
+				By("when cluster has > 2 nodes, network allocation condition should have an error")
 				found := false
 				for _, condition := range actualConditions {
 					if condition.Type == netAllocationCondition && condition.Status == metav1.ConditionFalse {
 						found = true
 					}
 				}
 				Expect(found).To(BeTrue(), "NetworkAllocationSucceeded condition should be False when cluster has > 2 nodes")
+				By("checking that a NetworkAllocationFailed error event was also reported")
 				events, err := cs.CoreV1().Events(f.Namespace.Name).List(context.Background(), metav1.ListOptions{})
 				Expect(err).NotTo(HaveOccurred())
 				found = false
 				for _, event := range events.Items {
 					if event.Reason == "NetworkAllocationFailed" && event.LastTimestamp.After(time.Now().Add(-30*time.Second)) &&
-						strings.Contains(event.Message, "error allocating network") {
+						strings.Contains(event.Message, "error allocating subnet") {
```

Comment:
this may need a change in origin? :)

Reply:
and thats enough for me to revert the message back tot he old one :)